src/routes/my/_Empty.svelte

```diff
@@ -1,40 +1,248 @@
+        fill="#D2D6E7"
+      />
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
+        d="M64.8602 100.863V128.881L52.7344 119.317L64.8602 100.863ZM53.6671 124.211L64.8602 132.617V141.956L53.6671 124.211Z"
+        fill="#FCFDFE"
+      />
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
+        d="M77.9189 119.513L64.7227 99.9294C60.2579 106.196 56.2668 112.786 51.8018 119.053L64.7019 129.162C66.7592 127.657 68.8188 126.155 70.8787 124.654C73.2267 122.942 75.5744 121.23 77.9189 119.513ZM64.2346 127.58L53.1099 118.862C56.9571 113.427 60.4242 107.734 64.2346 102.274V127.58Z"
+        fill="#D2D6E7"
+      />
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
+        d="M64.7159 142.89L77.2918 122.291L64.7088 131.798L51.8932 122.311L64.7159 142.89ZM55.3598 126.068L64.2346 132.638V140.311L55.3598 126.068Z"
+        fill="#D2D6E7"
+      />
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
+        d="M327.431 5.60352L327.897 41.0925L312.04 29.2507L327.431 5.60352ZM313.906 36.4229L327.897 46.696L327.897 58.837L313.906 36.4229Z"
+        fill="#FCFDFE"
+      />
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
+        d="M344.687 29.7353L327.714 4.66968C321.973 12.6882 316.848 21.1271 311.107 29.1453L327.694 42.0803C333.354 37.9598 339.028 33.8585 344.687 29.7353ZM327.242 40.5587L312.372 28.9626C317.516 21.7427 322.135 14.1683 327.242 6.92316V40.5587Z"
+        fill="#D2D6E7"
+      />
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
+        d="M327.707 59.771L343.512 34.0095L327.7 45.8979L311.589 34.0288L327.707 59.771ZM314.94 37.6426L327.242 46.7048V57.2888L314.94 37.6426Z"
+        fill="#D2D6E7"
+      />
+    </svg>
+  {/if}
```

Comment:
There are 2 ways to do it:
1. Put it in the separate `svg` file to reduce the bundle size and improve loading times
2. Only 1 `svg` sprite needed. It should be added to `webkit/illus`. This illustration should contain css variables